src/components/ProductCard.tsx

```diff
@@ -7,18 +7,13 @@ import ProductCardShowMoreButton from "./productCardShowMoreButton";
 export type Content = {
   colours: string[];
   image: string[];
-  showMoreImage: string;
+  showMoreImage: string[];
```

Comment:
I don't think the property name matches what it's doing. I would expect showMoreImage to be a boolean to dictate whether or not to show more images.

What's the expected behaviour change here? Originally I think the showMoreImage was to show the 2nd image on the back of the card?

Reply:
is supposed to show the second image of the care. Changed to more appropriate naming.